apps/docs/pages/guides/auth/auth-helpers/sveltekit.mdx

```diff
@@ -261,6 +261,7 @@ To utilize Supabase in shared load functions and within pages, it is essential t
 // src/routes/+layout.js
 import { PUBLIC_SUPABASE_ANON_KEY, PUBLIC_SUPABASE_URL } from '$env/static/public'
 import { createSupabaseLoadClient } from '@supabase/auth-helpers-sveltekit'
+import { PUBLIC_SUPABASE_ANON_KEY, PUBLIC_SUPABASE_URL } from '$env/static/public';
```

Comment:
Hey @danawoodman, a bit confused why this is imported twice?

Reply:
Hmm that's weird... I'm not sure what I was smoking but I could have sworn one of these examples was missing the import...